Sources/Datadog/RUM/RUMTelemetry.swift

```diff
@@ -5,6 +5,9 @@
  */
 
 import Foundation
+import UIKit
```

Comment:
I'll double check. I was curious about that too but just assumed it was because of something in the automatic view detection... but it's possible a typo caused it to auto import.